app/src/main/res/values/strings.xml

```diff
@@ -306,9 +316,29 @@ Please report any problems you encounter via Github.</string>
     <string name="sync_only_wifi_ssids_location_permission_rejected_dialog_title">Permission required</string>
     <string name="sync_only_wifi_ssids_location_permission_rejected_dialog_content">Starting with Android 8.1, location access is required to be able to read the WiFi\'s name. You can use this feature only if you grant this permission.</string>
 
+    <string name="power_source_title">Run when device powered by</string>
+
+    <string-array name="power_source_entries">
+        <item>AC and battery power</item>
+        <item>AC power</item>
+        <item>Battery power</item>
+    </string-array>
+
     <string name="respect_battery_saving_title">Respect Android battery saving setting</string>
     <string name="respect_battery_saving_summary">Disable Syncthing if battery saving is active</string>
 
+    <string name="respect_master_sync_title">Respect Android \'Auto-sync data\' setting</string>
+    <string name="respect_master_sync_summary">Disable Syncthing when the quick settings tile \'Auto-sync data\' is toggled off.</string>
+
+    <string name="run_in_flight_mode_title">Run when device is in flight mode</string>
+    <string name="run_in_flight_mode_summary">Enable if your phone has problems detecting manually connected wifi during flight mode.</string>
+
+    <string name="service_settings_title">Service setting</string>
+    <string name="service_settings_summary">Running Syncthing as a service adds a persistent notification to prevent it from being ended by android. Checking this option starts syncthing automatically when the phone boots.</string>
```

Comment:
Running Syncthing as a service. Adds a persistent notification to prevent it from being terminated by Android. Checking this option starts Syncthing on operating system startup.

Reply:
ok